### PR TITLE
chore: communities tests enabled and fixed, replaced image comparison

### PR DESCRIPTION
**Pull request by Valentina1133**
* Fixed join community test (object names changed in the app, also there were some changes in scenarios, so I updated test)
* Got rid of image comparison, in some tests I had to remove it at all (because we don't use screenshots anymore, but it was not possible to replace it with other verifications), in others - just replaced with another verifications (color, emoji etc.)
* Removed skip marks from all communities tests
* Commented one verification (will be enabled after bug is fixed)
* Changed method of getting members and name, because it didn't work anymore in old method

Test run on CI: https://ci.status.im/job/status-desktop/job/e2e/job/manual/1200/allure/

### CODE REVIEW

File: tests/communities/test_communities.py
Comment (glitchminer) [inline tests/communities/test_communities.py:1]: I'm getting an error when running locally. In `Verify community parameters in community settings screen` it seems to be opening the main app settings and then getting stuck.

```
    @allure.step('Wait until appears {0}')
    def wait_until_appears(self, timeout_msec: int = configs.timeouts.UI_LOAD_TIMEOUT_MSEC):
>       assert driver.waitFor(lambda: self.is_visible, timeout_msec), f'Object {self} is not visible'
E       AssertionError: Object SettingsScreen(mainWindow_ProfileLayout) is not visible
```
```
    with step('Verify community parameters in community settings screen'):
        settings_screen = main_screen.left_panel.open_settings()
        community_settings = settings_screen.left_panel.open_communities_settings()
```

Comment (glitchminer) [reply tests/communities/test_communities.py:1]: Issue with local config. resolved.

**Review by glitchminer — APPROVED** [review]

**Review by anastasiyaig — APPROVED** [review]

**Review by borismelnik — APPROVED** [review]
👍

**Comment by anastasiyaig** [comment]
@Valentina1133 i rebased into recent changes and it seems that my suspect regarding the ApplicationContext is relevant, once you moved the test with community invite into separate test, i am seeing it passes which kinda confirms my idea. Lets merge this PR

**Pull request merged**